pandas/core/array_algos/masked_reductions.py

```diff
@@ -100,3 +100,9 @@ def min(values: np.ndarray, mask: np.ndarray, skipna: bool = True):
 
 def max(values: np.ndarray, mask: np.ndarray, skipna: bool = True):
     return _minmax(np.max, values=values, mask=mask, skipna=skipna)
+
+
+def mean(values: np.ndarray, mask: np.ndarray, skipna: bool = True, min_count: int = 0):
```

Comment:
```suggestion
def mean(values: np.ndarray, mask: np.ndarray, skipna: bool = True):
```

The `mean` method has no `min_count` keyword in pandas, so that can be removed here.